sc-kpm/sc-agents-common/utils/IteratorUtils.cpp

```diff
@@ -33,7 +33,7 @@ ScAddr IteratorUtils::getFirstFromSet(ScMemoryContext * ms_context, const ScAddr
   }
   else
   {
-    ScIterator3Ptr iterator3 = ms_context->Iterator3(set, ScType::EdgeAccessConstPosPerm, ScType::Node);
+    ScIterator3Ptr iterator3 = ms_context->Iterator3(set, ScType::EdgeAccessConstPosPerm, ScType::Unknown);
```

Comment:
we have above:
ms_context->Iterator5(
          set,
          ScType::EdgeAccessConstPosPerm,
          ScType::Node,
          ScType::EdgeAccessConstPosPerm,
          scAgentsCommon::CoreKeynodes::rrel_1);
So, if get_strictly_first then we search for node else we search for node or arc/edge. I thin it's better to replace it above to.